### DISCUSSION TITLE
Индивидуальный код на каждую строку в таблице

### DISCUSSION THREAD

**Discussion by an220022** [totumonline/totum-ru-issues-and-discussions#69] [Как это сделать?] [answered]
Необходимо сделать формирование отчета за выбранный период по установленной форме через временную таблицу. Статьи отчета статичны, а суммы по каждой статье необходимо рассчитать, агрегировав данные с разных простых таблиц за выбранный период. Для расчета суммы **по каждой статье (строке) требуется прописать свой код.**

Как проще сделать такой отчет?

**Comment by an220022** [comment, marked as answer]
У меня была идея создать отдельную простую таблицу с 2 колонками: 1 колонка - Названия статей отчета, 2 колонка - код (текст). Строки временной таблицы заполнять через простую и использовать код простой таблицы через exec

**Reply by ttm-alexeypolunin** [reply]
Да, подтягиваешь кодом при добавлении код из простой и в последующих полях обращаешся через `exec`.
